packages/atomic/project.json

```diff
@@ -86,80 +65,27 @@
       }
     },
     "list-assets": {
```

Comment:
In general, I think these files would be easier to read if the targets that don't have dependencies were listed first.